### DISCUSSION TITLE
logstash导出数据到doris报错

### DISCUSSION THREAD

**Discussion by MrGreen1024** [apache/doris#7237] [A - General / Q&A]
根据官方文档，通过logstash导出数据到doris失败
logstash配置：
`input { stdin { } }

output {
    stdout {
		codec=>rubydebug{}
	}
	doris {
        http_hosts => [ "http://127.0.0.1:9030" ]
        user => doris
        password => xxx
        db => "cdn_check"
        table => "output"
	label_prefix => "doris"
        column_separator => ","
        columns => "a, b, c, d, e"
    }
}`
错误信息：
`Failed to flush outgoing items {:outgoing_count=>1, :exception=>"Net::HTTPBadResponse", :backtrace=>["uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/net/http/response.rb:42:in `read_status_line'", "uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/net/http/response.rb:29:in `read_new'", "uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/net/http.rb:1504:in `block in transport_request'", "org/jruby/RubyKernel.java:1189:in `catch'", "uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/net/http.rb:1501:in `transport_request'", "uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/net/http.rb:1474:in `request'", "D:/tools/logstash-7.15.2/vendor/bundle/jruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/request.rb:471:in `net_http_do_request'", "D:/tools/logstash-7.15.2/vendor/bundle/jruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/request.rb:733:in `block in transmit'", "uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/net/http.rb:914:in `start'", "D:/tools/logstash-7.15.2/vendor/bundle/jruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/request.rb:727:in `transmit'", "D:/tools/logstash-7.15.2/vendor/bundle/jruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/request.rb:163:in `execute'", "D:/tools/logstash-7.15.2/vendor/bundle/jruby/2.5.0/gems/rest-client-2.1.0/lib/restclient/request.rb:63:in `execute'", "D:/tools/logstash-7.15.2/vendor/bundle/jruby/2.5.0/gems/rest-client-2.1.0/lib/restclient.rb:78:in `put'", "D:/tools/logstash-7.15.2/vendor/local_gems/a69e8893/logstash-output-doris-0.1.0/lib/logstash/outputs/doris.rb:220:in `make_request'", "D:/tools/logstash-7.15.2/vendor/local_gems/a69e8893/logstash-output-doris-0.1.0/lib/logstash/outputs/doris.rb:189:in `flush'", "D:/tools/logstash-7.15.2/vendor/bundle/jruby/2.5.0/gems/stud-0.0.23/lib/stud/buffer.rb:219:in `block in buffer_flush'", "org/jruby/RubyHash.java:1415:in `each'", "D:/tools/logstash-7.15.2/vendor/bundle/jruby/2.5.0/gems/stud-0.0.23/lib/stud/buffer.rb:216:in `buffer_flush'", "D:/tools/logstash-7.15.2/vendor/bundle/jruby/2.5.0/gems/stud-0.0.23/lib/stud/buffer.rb:112:in `block in buffer_initialize'", "org/jruby/RubyKernel.java:1442:in `loop'", "D:/tools/logstash-7.15.2/vendor/bundle/jruby/2.5.0/gems/stud-0.0.23/lib/stud/buffer.rb:110:in `block in buffer_initialize'"]}`

**Comment by dataalive** [comment]
问题还在吗？
